web/src/main/java/se/inera/intyg/intygstjanst/web/service/repo/model/CitizenCertificate.java

```diff
@@ -19,15 +19,18 @@
 
 package se.inera.intyg.intygstjanst.web.service.repo.model;
 
+import java.time.LocalDateTime;
+import java.util.List;
+import lombok.AllArgsConstructor;
 import lombok.Builder;
 import lombok.Data;
+import lombok.NoArgsConstructor;
 import se.inera.intyg.intygstjanst.web.service.dto.citizen.CitizenCertificateRelationDTO;
 
-import java.time.LocalDateTime;
-import java.util.List;
-
 @Data
 @Builder
+@NoArgsConstructor
```

Comment:
Lets go with full on with builder and value :-) I know it was Data from before, but maybe we can refactor it to make things immutable.